src/dtm.jl

```diff
@@ -113,6 +193,26 @@ function dtm_entries(d::AbstractDocument, lex::Dict{String, Int})
     return (indices, values)
 end
 
+"""
+```
+dtv(d, lexicon)
+```
```

Comment:
This part doesn't seem consistent with the remaining docs, the official julia documentation says to use 4 spaced indent and then the function signature.